src/main/java/ro/fortsoft/pf4j/update/DefaultUpdateRepository.java

```diff
@@ -125,4 +134,71 @@ public String getPluginsJsonFileName() {
     public void setPluginsJsonFileName(String pluginsJsonFileName) {
         this.pluginsJsonFileName = pluginsJsonFileName;
     }
+
+    /* Fork of com.google.gson.internal.bind.DateTypeAdapter */
+    private static class LenientDateTypeAdapter extends TypeAdapter<Date> {
```

Comment:
Is it possible to move this inner class (in a file `LenientDateTypeAdapter.java`) outside? The reason is to reduce the lines number from `DefaultUpdateRepository` (better readability).